### ISSUE TITLE
Can't get .tox to load in a vanilla project

### ISSUE THREAD

**Issue by jetXS** [TouchDesigner/TouchEngine-UE4#90]
Started UE 4.26.2 and selected to create a new project of type "Virtual Production"
Didn't select started content
Didn't select ray tracing

Added plugin to folder, enabled plugin, restarted project

In this new "default" project without changes, I created from the content browser a TouchEngine Actor.

Opened the BP of that new Actor, proceeded to load a .tox... got a crash.

Added a TouchEngine shortcut next to my .tox just to make sure

Went back to editing the BP with the .tox path, got a crash again.

```
Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0xffffffffffffffff

UE4Editor_Core
UE4Editor_Core
UE4Editor_Core
UE4Editor_Core
UE4Editor_Core
UE4Editor_TouchEngineEditor!TArray<wchar_t,TSizedDefaultAllocator<32> >::ResizeTo() [C:\Program Files\Epic Games\UE_4.26\Engine\Source\Runtime\Core\Public\Containers\Array.h:2580]
UE4Editor_TouchEngineEditor!FString::operator=<FString &,0>() [C:\Program Files\Epic Games\UE_4.26\Engine\Source\Runtime\Core\Public\Containers\UnrealString.h:307]
UE4Editor_TouchEngineEditor!FTouchEngineDynamicVariableStructDetailsCustomization::ToxFailedLoad() [H:\_DERIVATIVE\Github\TouchEngine-UE4-Plugin-Release\TEMP\TouchEngine-UE4_0.9.2\TouchEngine-UE4\HostProject\Plugins\TouchEngine\Source\TouchEngineEditor\Private\TouchEngineDynVarDetsCust.cpp:782]
UE4Editor_TouchEngineEditor!TBaseRawMethodDelegateInstance<0,FTouchEngineDynamicVariableStructDetailsCustomization,void __cdecl(FString),FDefaultDelegateUserPolicy>::ExecuteIfSafe() [C:\Program Files\Epic Games\UE_4.26\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:467]
UE4Editor_TouchEngine!TMulticastDelegate<void __cdecl(FString),FDefaultDelegateUserPolicy>::Broadcast() [C:\Program Files\Epic Games\UE_4.26\Engine\Source\Runtime\Core\Public\Delegates\DelegateSignatureImpl.inl:955]
UE4Editor_TouchEngine!UTouchEngineComponentBase::PostEditChangeProperty() [H:\_DERIVATIVE\Github\TouchEngine-UE4-Plugin-Release\TEMP\TouchEngine-UE4_0.9.2\TouchEngine-UE4\HostProject\Plugins\TouchEngine\Source\TouchEngine\Private\TouchEngineComponent.cpp:201]
UE4Editor_CoreUObject
UE4Editor_Engine
UE4Editor_CoreUObject
UE4Editor_CoreUObject
UE4Editor_Engine
UE4Editor_PropertyEditor
UE4Editor_PropertyEditor
UE4Editor_PropertyEditor
UE4Editor_PropertyEditor
UE4Editor_PropertyEditor
UE4Editor_PropertyEditor
UE4Editor_Slate
UE4Editor_Slate
UE4Editor_Slate
UE4Editor_Slate
UE4Editor_Slate
UE4Editor_Slate
UE4Editor_Slate
UE4Editor_Slate
UE4Editor_ApplicationCore
UE4Editor_ApplicationCore
UE4Editor_ApplicationCore
UE4Editor_ApplicationCore
user32
user32
UE4Editor_ApplicationCore
UE4Editor
UE4Editor
UE4Editor
UE4Editor
UE4Editor
kernel32
ntdll
```

**Comment by jetXS** [comment]
Blocked by #98

**Comment by ddthejj** [comment]
This has been fixed in ded5cb659b9a6eb5ef60d1de56bb1a5594c844e7

**Comment by jetXS** [comment]
Tested and confirmed as fixed.
